src/v3/src/img/16pxDevice.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="desktop" role="img">
```

Comment:
The aria-labelled by values should match the `id` below in `title` -- I think these should fail a11y checks if we check a view that has this SVG